python/nrel/routee/compass/resources/osm_default_energy.toml

```diff
@@ -1,6 +1,12 @@
 parallelism = 2
 search_orientation = "vertex"
 
+[state]
+distance = { distance_unit = "miles", initial = 0.0 }
+time = { time_unit = "minutes", initial = 0.0 }
+energy_liquid = { energy_unit = "gallons_gasoline_per_mile", initial = 0.0 }
```

Comment:
I think the energy unit is "gallons_gasoline"

Reply:
lol yup i know i mixed that up :-D